eWonConnector-gateway/src/main/java/com/hms_networks/americas/sc/ignition/comm/AsyncHttpRequestManager.java

```diff
@@ -70,14 +111,43 @@ public static void initialize(boolean isDebugEnabled) {
     if (isNotInitialized()) {
       LOGGER.info("Initializing Asynchronous HTTP manager...");
 
-      // Initialize the asynchronous HTTP client
-      RequestConfig requestConfig =
-          RequestConfig.custom()
-              .setConnectionRequestTimeout(CONNECT_TIMEOUT_MILLIS, TimeUnit.MILLISECONDS)
-              .setResponseTimeout(SOCKET_TIMEOUT_MILLIS, TimeUnit.MILLISECONDS)
-              .build();
-      httpAsyncClient = HttpAsyncClients.custom().setDefaultRequestConfig(requestConfig).build();
-      httpAsyncClient.start();
+      // Create connection manager
+      try {
+        asyncClientConnectionManager = new PoolingAsyncClientConnectionManager();
+        asyncClientConnectionManager.setMaxTotal(HTTP_CXN_POOL_MAX_REQUESTS);
```

Comment:
This seems like a lot. Could be memory intensive.

Reply:
This is necessary to buffer requests and, similar to the Cxn request timeout, is large enough to support accounts with many Ewons.

The memory intensity would be related/tied to the number of Ewons on the account, which influences the number of necessary requests.